hack/generators/release-controllers/content/osd_rc_deployments.py

```diff
@@ -226,7 +226,7 @@ def _add_osd_rc_deployment(gendoc):
 
     # Creating Cluster Groups for the AMD64 jobs...
     if context.arch == 'x86_64':
-        extra_rc_args.append('--cluster-group=build01,build03,build04,build05')
+        extra_rc_args.append('--cluster-group=build02,build03,build04')
```

Comment:
We probably dont want to revert this change completely, but simply add back `build02`.

Reply:
Ack. Let me revise.

Reply:
Done!